README.md

```diff
@@ -3,11 +3,11 @@ cibuildwheel
 
 [![PyPI](https://img.shields.io/pypi/v/cibuildwheel.svg)](https://pypi.python.org/pypi/cibuildwheel)
 [![Documentation Status](https://readthedocs.org/projects/cibuildwheel/badge/?version=stable)](https://cibuildwheel.readthedocs.io/en/stable/?badge=stable)
-[![Actions Status](https://github.com/joerick/cibuildwheel/workflows/Test/badge.svg)](https://github.com/joerick/cibuildwheel/actions)
-[![Travis Status](https://img.shields.io/travis/joerick/cibuildwheel/master?logo=travis)](https://travis-ci.org/joerick/cibuildwheel)
-[![AppVeyor status](https://ci.appveyor.com/api/projects/status/wbsgxshp05tt1tif/branch/master?svg=true)](https://ci.appveyor.com/project/joerick/cibuildwheel/branch/master)
-[![CircleCI Status](https://img.shields.io/circleci/build/gh/joerick/cibuildwheel/master?logo=circleci)](https://circleci.com/gh/joerick/cibuildwheel)
-[![Azure Status](https://dev.azure.com/joerick0429/cibuildwheel/_apis/build/status/joerick.cibuildwheel?branchName=master)](https://dev.azure.com/joerick0429/cibuildwheel/_build/latest?definitionId=2&branchName=master)
+[![Actions Status](https://github.com/pypa/cibuildwheel/workflows/Test/badge.svg)](https://github.com/pypa/cibuildwheel/actions)
+[![Travis Status](https://img.shields.io/travis/com/pypa/cibuildwheel/master?logo=travis)](https://travis-ci.com/pypa/cibuildwheel)
+[![Appveyor status](https://ci.appveyor.com/api/projects/status/gt3vwl88yt0y3hur/branch/master?svg=true)](https://ci.appveyor.com/project/joerick/cibuildwheel/branch/master)
+[![CircleCI Status](https://img.shields.io/circleci/build/gh/pypa/cibuildwheel/master?logo=circleci)](https://circleci.com/gh/pypa/cibuildwheel)
+[![Azure Status](https://dev.azure.com/joerick0429/cibuildwheel/_apis/build/status/pypa.cibuildwheel?branchName=master)](https://dev.azure.com/joerick0429/cibuildwheel/_build/latest?definitionId=4&branchName=master)
```

Comment:
@joerick,
 
It seems a bit odd that AppVeyor & Azure are still referencing your account.
I know it's working so that's just a minor remark but if you have a bit more insight on this, I'd like to learn something.

Reply:
Yeah, it's a little strange - I'm not sure how Appveyor is defining their data model, but it seems that I create a project within 'joerick' even though I link it with an org project. Azure seems even more disconnected, I create an Azure project, then tie it to any CI system, so you could also create an Azure project that's driven from the same codebase. The link back to each is defined by the Webhooks in Github, so that's where the definitive list of CIs on this project is stored - https://github.com/pypa/cibuildwheel/settings/hooks

Reply:
Thanks for the insight.
From the webhooks list you link to, gitlab seems to throw some errors. As it was working yesterday, should we consider this a temporary glitch and merge this or is there something more to look at ?

Reply:
I think it's fine to squash & merge, and I could start the backport to 1.x (which likely will also need some of the CI changes). If GitLab has issues we can open a new PR.